CHANGES.rst

```diff
@@ -257,6 +257,8 @@ Bug Fixes
 
 - ``astropy.table``
 
+  - Fix reading of big ascii tables with the fast reader. [#5319]
```

Comment:
Right, the limit is 2Gb, as the code used signed int. 
I updated the changelog, and moved it to 1.0. Should I duplicate the line in the 1.2 section ?